alerting/src/main/kotlin/org/opensearch/alerting/util/destinationmigration/NotificationApiUtils.kt

```diff
@@ -138,33 +136,11 @@ suspend fun NotificationConfigInfo.sendNotification(client: Client, title: Strin
 }
 
 /**
- * For most channel types, a placeholder Alerting title will be used but the email channel/SNS notification will
- * use the subject, so it appears as the actual subject of the email/SNS notification.
+ * A placeholder Alerting title will be used if no subject is passed in.
  */
 fun NotificationConfigInfo.getTitle(subject: String?): String {
     val defaultTitle = "Alerting-Notification Action"
-    if (this.notificationConfig.configType == ConfigType.EMAIL || this.notificationConfig.configType == ConfigType.SNS) {
-        return if (subject.isNullOrEmpty()) defaultTitle else subject
-    }
-
-    return defaultTitle
-}
-
-fun NotificationConfigInfo.createMessageContent(subject: String?, message: String): String {
-    // For Email Channels, the subject is not passed in the main message since it's used as the title
-    if (this.notificationConfig.configType == ConfigType.EMAIL) {
-        return constructMessageContent("", message)
-    }
-
-    return constructMessageContent(subject, message)
-}
-
-/**
- * Similar to Destinations, this is a generic utility method for constructing message content from
- * a subject and message body when sending through Notifications since the Action definition in Monitors can have both.
- */
-private fun constructMessageContent(subject: String?, message: String): String {
-    return if (Strings.isNullOrEmpty(subject)) message else "$subject \n\n $message"
+    return if (subject.isNullOrEmpty()) defaultTitle else subject
```

Comment:
Title and subject are not necessarily interchangeable right? Also it seems we've now removed the `constructMessageContent` method, is this being replaced by a similar call somewhere else?

Reply:
I was looking into the behavior on Notification and subject is like the title being used here and I think it was a bug for alerting to think that subject wouldn't equal title. This goes back to how Alerting behaved before 2.0. The Notification plugin tried to be the same as Alerting's notification sub project, so the title there behaves similarly to subject.